packages/web-frontend/src/containers/EnlargedDialog/EnlargedDialog.js

```diff
@@ -162,8 +162,8 @@ const EnlargedDialogComponent = ({
 
 EnlargedDialogComponent.propTypes = {
   onCloseOverlay: PropTypes.func.isRequired,
-  viewContent: PropTypes.shape(VIEW_CONTENT_SHAPE).isRequired,
```

Comment:
Happy for pushback on this but seems like these props are passed to `EnlargedDialogContent` which are unrequired. Where `EnlargedDialogComponent` uses viewContent it tests for it's existence. The console errors of missing proptypes seem to occur in a preload of component.